spec/models/ext_management_system_spec.rb

```diff
@@ -74,7 +76,7 @@
     it "permissions.tmpl.yml should contain all EMS types" do
       types = YAML.load_file(Rails.root.join("config/permissions.tmpl.yml"))
       stub_vmdb_permission_store_with_types(types) do
-        expect(described_class.supported_types_and_descriptions_hash).to eq(all_types_and_descriptions)
+        expect(described_class.supported_types_and_descriptions_hash).to eq(all_types_and_descriptions.reject { |type, _desc| downstream_unsupported_types.include?(type) })
```

Comment:
@agrare this condition seems suspicious. I thought there were more downstream-disabled providers but all others are present in yaml above